src/ApiTypeDocument.js

```diff
@@ -291,6 +293,8 @@ export class ApiTypeDocument extends PropertyDocumentMixin(LitElement) {
     this.narrow = false;
     this.selectedBodyId = undefined;
     this.aware = undefined;
+
+    this._isPropertyReadOnly = this._isPropertyReadOnly.bind(this);
```

Comment:
Why this binding here? It seems not to be used in any event.

Reply:
I thought the same, but I was having a context error otherwise